app/models/illumina_htp/initial_stock_tube_purpose.rb

```diff
@@ -22,9 +22,13 @@ def sibling_tubes(tube) # rubocop:todo Metrics/AbcSize
     sibling_requests = tube.submission.requests.multiplexed.opened.ids
 
     sibling_tubes =
+      # rubocop:todo Layout/LineLength
       Tube.joins(:transfer_requests_as_target).includes(:transfer_requests_as_source) # Outer join, as we don't want these
+        # rubocop:enable Layout/LineLength
         .where(transfer_requests: { submission_id: tube.submission }) # find out tubes via transfer requests
+        # rubocop:todo Layout/LineLength
         .where("transfer_requests_as_sources_#{Tube.table_name}": { id: nil }) # Make sure we have no transfers out of the tube
+        # rubocop:enable Layout/LineLength
```

Comment:
I know I said I wouldn't comment on these any more, but this one is particularly frustrating.